Modules/UDP/.gitignore

```diff
@@ -0,0 +1,31 @@
+# Created by https://www.gitignore.io/api/xcode
```

Comment:
could you use the one that SPM generates? https://github.com/Zewo/Zewo/blob/master/.gitignore